### PR TITLE
Added ability for the app to start on login

### PR DESCRIPTION
**Pull request by rareinator**

### CODE REVIEW

File: Chronos/AppDelegate.swift [hunk 1]
```diff
@@ -16,6 +17,19 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     }()
 
     func applicationDidFinishLaunching(_ notification: Notification) {
+        // Checking to see if the Launcher application helper app is already running
+        // if it is we kill it
+        let launcherAppIdentifier = "nogen.Chronus.LauncherApplication"
```
Comment (qruzz) [inline Chronos/AppDelegate.swift]: There is a typo in `Chronus` - `u` should be replaced with `o`

File: Chronos/AppDelegate.swift [hunk 2]
```diff
@@ -16,6 +17,19 @@ class AppDelegate: NSObject, NSApplicationDelegate {
     }()
 
     func applicationDidFinishLaunching(_ notification: Notification) {
+        // Checking to see if the Launcher application helper app is already running
+        // if it is we kill it
```
Comment (qruzz) [inline Chronos/AppDelegate.swift]: This comment belongs above line 29 right I think

File: LauncherApplication/AppDelegate.swift [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            NSWorkspace.shared.openApplication(at: url!, configuration: configuration, completionHandler: nil)
+        } else {
+            // If its already running we just kill the main application
+            terminateApplication()
+        }
+    }
+
+    func applicationWillTerminate(_ aNotification: Notification) {
+        // Insert code here to tear down your application
+    }
```
Comment (qruzz) [inline LauncherApplication/AppDelegate.swift]: I would just remove this?
Or is it required to be there when you extend an `NSApplicationDelegate`?

**Review by qruzz — COMMENTED** [review]
Looks really good all-in-all - just a couple tiny comments 👍

**Review by qruzz — DISMISSED** [review]
The comment _"... telling it to kill itself"_ .. not sure whether we should rephrase that as it sounds rather extreme 😅
Otherwise it looks really good - nice job 👍

**Comment by rareinator** [comment]
> The comment _"... telling it to kill itself"_ .. not sure whether we should rephrase that as it sounds rather extreme 😅
> 
> Otherwise it looks really good - nice job 👍 

I mean we can change it to terminate, they mean the same thing 🙂

**Review by qruzz — APPROVED** [review]

**Pull request merged**